tests/test-vip-cache-manager.php

```diff
@@ -66,12 +77,14 @@ public function test__valid__queue_purge_url( $queue_url, $expected_urls ) {
 	 * @dataProvider get_data_for_invalid_queue_purge_url_test
 	 */
 	public function test__invalid__queue_purge_url( $queue_url ) {
-		$this->expectWarning();
-
-		$actual_output = $this->cache_manager->queue_purge_url( $queue_url );
-
-		$this->assertFalse( $actual_output, 'Return value from `queue_purge_url` should be false.' );
-		$this->assertEmpty( $this->cache_manager->get_queued_purge_urls(), 'List of queued purge urls should be empty' );
```

Comment:
When we `expectWarning()`/`expectException()`/…, the code that follows the statement that is supposed to pit a warning or throw an exception, does not run. Therefore, if we really need to run some checks, we have to catch the exception ourselves.

Reply:
See #4464